index.js

```diff
@@ -1,14 +1,29 @@
 function calculate(numberOne, operation, numberTwo) {
+  if (numberOne === undefined) {
+    numberOne = 0
+  }
+  if (numberTwo === undefined) {
+    numberTwo = 0
+  }
+
   switch (operation) {
     case '+':
-      return numberOne - numberTwo
+      return parseInt(numberOne) + parseInt(numberTwo)
     case '-':
-      return numberOne + parseInt(numberTwo)
+      return parseInt(numberOne) - parseInt(numberTwo)
+    case '*':
+      return parseInt(numberOne) * parseInt(numberTwo)
     default:
       return 'Operation not yet supported. You sent ' + operation
   }
 }
 
+// console logs to help visual it's working
```

Comment:
Nicely done and nice comment. 
The idea of the test files is to provide the visual easily as well but it's tough to get simpler and easier than console.log so nice additional add. In a production app we would want to remove the the console.log before we commit to git.